src/module_wrap.cc

```diff
@@ -173,9 +177,11 @@ void ModuleWrap::Link(const FunctionCallbackInfo<Value>& args) {
     }
     Local<Promise> resolve_promise = resolve_return_value.As<Promise>();
     obj->resolve_cache_[specifier_std].Reset(env->isolate(), resolve_promise);
+
+    USE(promises->Set(mod_context, specifier, resolve_promise));
```

Comment:
This *should* crash if the Set fails (`promises->Set().FromJust()`). USE is only for when we don’t care about the result of execution, and should be used sparingly.